conf/ocsci/openshift_dedicated.yaml

```diff
@@ -10,8 +10,10 @@ ENV_DATA:
   skip_ocs_deployment: true
 
 # AUTH:
-#  openshiftdedicated:
-#    token: OCM TOKEN KEY PLACEHOLDER
+#   pagerduty:
+#     api_key: PAGERDUTY API KEY PLACEHOLDER FOR MANAGED SERVICE
```

Comment:
I believe this needs documentation update on how to get this key

Reply:
There is a link to PagerDuty documentation in PagerDuty class that uses this token: https://github.com/red-hat-storage/ocs-ci/pull/4679/files#diff-8fb153a374c4dc7e74cec8d8619c5ea60c92a8e30ae157dde12209ade3a441aeR77